utils/collect.ts

```diff
@@ -136,9 +136,14 @@ const queryNominatim = async (lat: number, lon: number): Promise<NominatimRespon
 
         const nominatim = await queryNominatim(r.lat, r.lon);
 
+        let direction = parseInt(r.tags["camera:direction"]);
+        if (Number.isNaN(direction)) {
+            direction = parseInt(r.tags["direction"]);
+        }
```

Comment:
```suggestion
const direction = parseInt(r.tags["camera:direction"]) ||  parseInt(r.tags["direction"]) || undefined;
```

I like this syntax more because it removes the if statement, and you can read it better.

this suggestion needs to be tested

Reply:
This doesn't work because the direction may be 0 (i.e. north). I don't know if Typescript has an elegant way of expressing this operation.

Reply:
you are absolutely right I forgot about the falsyness of 0.

so js has the Nullish coalescing operator (??)
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

Reply:
But `NaN ?? undefined = NaN`. I think yet another operator is needed here ;)